tests/translate/translate_xppm.py

```diff
@@ -33,7 +33,7 @@ def process_inputs(self, inputs):
 
     def compute(self, inputs):
         self.process_inputs(inputs)
-        inputs["xflux"] = utils.make_storage_from_shape(inputs["q"].shape, xppm.origin)
+        inputs["xflux"] = utils.make_storage_from_shape(inputs["q"].shape, origin=(2, 0, 0))
```

Comment:
I prefer if we use the grid variable names rather than hard coded numbers, e.g. (grid.is_ - 1, grid.jsd, 0). We have started using harded coded 'add' tuples, which could be used as well (note there is an open PR changing the names of some of the grid shorthands)